Lamina/include/GL/window.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef LM_WINDOW
+#define LM_WINDOW
+
+#include "../Lamina/include/GL/colors.hpp"
+#include "../Lamina/include/Math/vertex.hpp"
+#include <glad/glad.h>
+#include <GLFW/include/GLFW/glfw3.h>
+#include <iostream>
+#include <string>
+
+//define lamina window types
```

Comment:
It's pretty clear that values relating to windows types are defined below so this comment doesn't add any additional information and should be removed. Also, as with my earlier comment, it's best to use real variables or enums for this purpose in C++.